app/controllers/state_file/questions/landing_page_controller.rb

```diff
@@ -2,6 +2,11 @@ module StateFile
   module Questions
     class LandingPageController < QuestionsController
       include StartIntakeConcern
+      before_action :before_update, only: :update
+
+      def before_update
+        sign_out current_intake if current_intake
+      end
```

Comment:
```ruby

    class LandingPageController < QuestionsController
      include StartIntakeConcern
      before_action :sign_out_old_intakes, only: :update

      def sign_out_old_intakes
        sign_out current_state_file_ny_intake if current_state_file_ny_intake.present?
        sign_out current_state_file_az_intake if current_state_file_az_intake.present?
      end
      
    end
```

Reply:
also for sign in and clearing session:
https://github.com/codeforamerica/vita-min/blob/0fb5189d03a1e0c5ec144173057a1a8656661d31/app/controllers/application_controller.rb#L70-L79